hayeon/programmers-0/정수를-나선형으로-배치하기.js

```diff
@@ -0,0 +1,48 @@
+function solution(n) {
+  const answer = [];
+
+  for (let i = 0; i < n; i++) {
+    answer.push(new Array(n).fill(0));
+  }
+
+  let num = 2;
+  let x = 0;
+  let y = 0;
+  answer[y][x] = 1;
```

Comment:
```suggestion
  const answer = Array.from({ length: n }, () => Array(n).fill(0));
  let num = 2;
  let x = 0;
  let y = 0;
```
- 위와 같이 for문없이 Array.from을 활용하는 방법도 있습니다
- 나머지 풀이 잘봣씁니다 ~ 다시한번 이해해보면 좋을 것 같습니다!

Reply:
- 오오 for문보다 훨씬 좋네요...!! 감사합니다 🙇‍♀️ 이 문제는 다시 풀어보려고 저장해놨습니다..🥲